src/core/Akka.Remote/Transport/DotNetty/BatchWriter.cs

```diff
@@ -82,13 +83,22 @@ public BatchWriterSettings(Config hocon)
     /// </summary>
     internal class BatchWriter : ChannelHandlerAdapter
     {
+        private class FlushCmd
+        {
+            public static readonly FlushCmd Instance = new FlushCmd();
+            private FlushCmd() { }
+        }
+
         public readonly BatchWriterSettings Settings;
+        public readonly IScheduler Scheduler;
+        private ICancelable _flushSchedule;
 
         internal bool CanSchedule { get; private set; } = true;
```

Comment:
It looks like we don't use `CanSchedule` anywhere, we are only setting it. Should it be removed?

Reply:
I'll remove it